Costumer.py

```diff
@@ -0,0 +1,17 @@
+class Costumer(object):
+    def __init__(self, _id, password, balance):
+        self.id = _id
+        self.password = password
+        self.balance = balance
+
+    def set_password(self, new_password):
+        self.password = new_password
+
+    def get_balance(self):
+        return self.balance
+
+    def set_balance(self, updated_bal):
+        self.balance = updated_bal

```

Comment:
Your code treats the balance as a string and converts it to a number every time you want to use it. Better to  store it as a number